plugins/postgres/dbt/include/postgres/macros/utils/columns_spec_ddl.sql

```diff
@@ -0,0 +1,23 @@
+{% macro postgres__get_columns_spec_ddl() %}
```

Comment:
Yeah, this was a toughie to discern. I biased towards the global macro default bundling the data type DDL and wrapping the select statement because other adapters may benefit from it(think: bigquery, snowflake, etc.). At the same time, the `postgres` implementation will likely be the most universally applicable minus the `check` constraint which isn't common across data warehouses. Overall, there is no good default implementation, just good template code to override in a specific adapter. 

**I recommend keeping it as is given the above, but given your team maintains this and you have a bigger picture view of how adapters will use this, I'm happy to make the `postgres` implementation the default.**